.github/workflows/tag.yml

```diff
@@ -1,58 +0,0 @@
-name: Tag
```

Comment:
Slight nit: this comes from years of managing yml. I personally find .yml files unreadable whenever you have any yml file above 100 lines. There is some opportunity to maybe split the .ymls and steps, but I think this is fine for now and dependent on personal opinion.

Why do you think we should have merged the files in your opinion?